packages/braid-design-system/src/lib/components/Accordion/AccordionContext.ts

```diff
@@ -1,10 +1,11 @@
 import { createContext } from 'react';
 import type { TextProps } from '../Text/Text';
 
+export type validSizeValues = TextProps['size'];
```

Comment:
Can we make types title case please? Also if we are not whitelisting a subset of sizes, maybe just use `TextProps['size']` without needing this separate type def. Thoughts?

Reply:
I just split it out to make the Record a bit cleaner in `Accordion.tsx`. But thinking about it, it might be ok to just use `TextProps['size']` there